src/core/level.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * This module updates the title and shortName
+ * of a spec when there is a valid level configuration.
+ *
+ * Levels should be integers >= 0.
+ */
+
+import { getIntlData, showInlineWarning } from "./utils.js";
+export const name = "core/level";
+
+const localizationStrings = {
+  en: {
+    level: "Level",
+  },
+};
+
+const l10n = getIntlData(localizationStrings);
+
+export function run(conf) {
+  const h1Elem = document.querySelector("h1#title");
+
+  if (conf.hasOwnProperty("level")) {
+    if (Number.isInteger(conf.level) && conf.level >= 0) {
```

Comment:
I think we should invert this one, so we do the validity check, and it fails, we `showInlineWarning()` and return. Finally, if all is good, we do the append. That will get rid of the else block too.

Reply:
we currently warn on the h1Elem though. would you prefer we warn on another element? `showInlineWarning()` needs an elem.

Reply:
Ah, good point! So this indeed comes after the first check. Flipping the if statement to get rid of the `else` would be good though.

Reply:
kk! as per MDN, `isInteger` returns ["A Boolean indicating whether or not the given value is an integer"](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isInteger),
 so I'm gonna adjust your code for that.